### PR TITLE
Changed layout of explore page

### LINKED ISSUE #74: Better UI for book list in the explore page
**Describe the bug**
When you hover over any book displayed in the explore page, 2 buttons (More Details, Rate this book) appear below the book div. This causes all the divs under this block to be shifted downwards, which appears to be messy and confusing.

**To Reproduce**
1. Go to https://kitabe-app.herokuapp.com/
2. Click on the explore button in the top right corner
3. Hover over any book and see how it changes when you unhover. 

**Expected behavior**
We can keep the layout consistent and change the divs in the explore page so that they are similar to the book list appearing in any genre page. For example: https://kitabe-app.herokuapp.com/genre_books/Fantasy 

**Screenshots**
image

The buttons that have appeared under the first book make the page look messy.

It could be made similar to the books in genre list
image

**Are you willing to contribute to this issue?** 
Yes. I would like to contribute to this issue as a Gssoc'21 participant.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: chrome
 - Version: 89

**Smartphone (please complete the following information):**
 - Device: Redmi K20

**Additional context**

### PR DESCRIPTION
**Pull request by AmrutaKoshe**
## Description
- Changed the layout of the explore page to make it user friendly. Made it similar to the layout of the book-genre page.
- made changes in the files explore.html and explore.css
- closes #74 

Before:
image

After:
image

on small screens:
explorePagePhone

## How has this been tested?
Ran it in my localhost

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/Praful932/Kitabe/blob/master/CONTRIBUTING.md).
- [x] My changes do not edit/add any unrequired files.
- [x] My changes are covered by tests.

### CODE REVIEW

**Review by maneprajakta — CHANGES_REQUESTED** [review]
- For large screens button seem to overlap
Screenshot from 2021-03-19 09-09-09
- For small screen content is not at center 
- Check this for small screen .
Screenshot from 2021-03-19 09-10-50

**Comment by AmrutaKoshe** [comment]
> * For large screens button seem to overlap
>   Screenshot from 2021-03-19 09-09-09
> * For small screen content is not at center
> * Check this for small screen .
>   Screenshot from 2021-03-19 09-10-50



> * For large screens button seem to overlap
>   Screenshot from 2021-03-19 09-09-09
> * For small screen content is not at center
> * Check this for small screen .
>   Screenshot from 2021-03-19 09-10-50

I have solved all these issues. Please review ~

**Comment by maneprajakta** [comment]
@AmrutaKoshe 
- Buttons still seem to overlap for large screens
- For small screen book card is not at the center now.

**Comment by maneprajakta** [comment]
@AmrutaKoshe 
Still, it doesn't seem to work :
- Because the card is not at the center the carousel control for the right side is not visible properly.
Screenshot from 2021-03-19 16-12-50
- Button overlapping problem is solved but on hover, it persists. Move rate this button left side.
k

**Comment by AmrutaKoshe** [comment]
> @AmrutaKoshe
> Still, it doesn't seem to work :
> 
> * Because the card is not at the center the carousel control for the right side is not visible properly.
>   Screenshot from 2021-03-19 16-12-50
> * Button overlapping problem is solved but on hover, it persists. Move rate this button left side.
>   k

Okay! I increased the gap between the buttons. 
For the cards in small screen, margin: auto doesn't work for some reason so I specified different margins with media queries. Does it look proper now?

**Comment by Praful932** [comment]
Hi @AmrutaKoshe , you can check out the changes in the temp deploy :)
https://kitabe-pipeline-pr-120.herokuapp.com/

**Comment by AmrutaKoshe** [comment]
> Hi @AmrutaKoshe , you can check out the changes in the temp deploy :)
> https://kitabe-pipeline-pr-120.herokuapp.com/

Yes! I have been checking that 😁 
And it looks fine on my device

File: static/mainapp/css/explore.css [hunk 1]
```diff
@@ -212,3 +212,15 @@
...
+
+@media (max-width: 370px){
+  .card, .card-deck{ 
+    margin-left: -1%;
+  }
+}
+
+@media (max-width: 390px){
+  .card, .card-deck{ 
+    margin-left: -0.5%;
```
Comment (maneprajakta) [inline static/mainapp/css/explore.css:224]: -1.5% works for aligning center

File: static/mainapp/css/explore.css [hunk 2]
```diff
@@ -212,3 +212,15 @@
     margin-bottom: 5%;
   }
 }
+
+@media (max-width: 370px){
```
Comment (maneprajakta) [inline static/mainapp/css/explore.css:216]: This query will not work as already 370px query is there

**Review by maneprajakta — CHANGES_REQUESTED** [review]
For large screen, if you could add bottom padding to the author name so that it doesn't collide with the buttons.
Screenshot from 2021-03-19 22-17-01

**Comment by AmrutaKoshe** [comment]
> For large screen, if you could add bottom padding to the author name so that it doesn't collide with the buttons.
> Screenshot from 2021-03-19 22-17-01

Okay! changed it to -1.5%
Also, the space between author name and buttons depends on how big the name of book & author is -
image

So I think a different issue can be created to resolve this problem !

**Review by maneprajakta — APPROVED** [review]

**Comment by maneprajakta** [comment]
@AmrutaKoshe Great  Work :rocket: 
Yep, a new issue needs to be created for that.

**Review by Praful932 — APPROVED** [review]
@AmrutaKoshe Lgtm, Impressive work 💯

**Pull request merged**